src/py/_pyodide/_core_docs.py

```diff
@@ -280,7 +280,7 @@ def _into_file(self, file: IOBase):
         data.
         """
 
-    def to_string(self, encoding=None) -> str:
```

Comment:
This is one endpoint I would have maybe expected to work with kwarg, but yes, OK if that's what we have now.

Reply:
Actually we *do* support encoding with keyword args:
```C
  static const char* const _keywords[] = { "encoding", 0 };
  static struct _PyArg_Parser _parser = { "|s:to_string", _keywords, 0 };
```
so this should be reverted.